README.md

```diff
@@ -1,2 +1,13 @@
 # workflows
 A set of example Argo workflows.
+
+
+## Provisioning
+
+To provision Argo, we use the [Argo Workflows ansible role](https://github.com/CermakM/ansible-role-argo-workflows) and the [provisioning playbook](/playbooks/provision.yml).
+
+You will be asked a namespace to with the Argo should be provisioned and a password to the vault. Run:
+
+```bash
+ansible-playbook playbooks/provision.yml --ask-vault-pass
```

Comment:
can we run this in a thoth-ops container? it should have all the modules required...